examples/zero-nextjs/next.config.js

```diff
@@ -0,0 +1,31 @@
+const { withZeroPlugin } = require('@mui/zero-next-plugin');
+const { extendTheme } = require('@mui/zero-runtime');
+
+// To learn more about theming, visit https://github.com/mui/material-ui/blob/master/packages/zero-runtime/README.md#theming
+const theme = extendTheme({
+  colorSchemes: {
+    light: {
+      palette: {
+        background: '0 0% 100%',
+        foreground: '240 10% 3.9%',
+        primary: '240 5.9% 10%',
+        border: '240 5.9% 90%',
+      },
+    },
+    dark: {
+      palette: {
+        background: '240 10% 3.9%',
+        foreground: '0 0% 80%',
+        primary: '0 0% 98%',
+        border: '240 3.7% 15.9%',
+      },
+    },
+  },
+});
+
+module.exports = withZeroPlugin(
+  {
+    // Your Next.js config
+  },
```

Comment:
This should be assigned to a `const nextConfig = {}` variable and then passed as first argument. Then you don't need the comment explanation.